From e5a4941d132080ac2a20aaf8659ce037ea6c4a3f Mon Sep 17 00:00:00 2001
From: Paul Gortmaker <pgortmak@windriver.com>
Date: Tue, 13 Nov 2007 21:33:54 +0000
Subject: [PATCH] kbuild: create a no-op rule for empty modules install

Add a rule relating to modules install that is a no-op when modules are
disabled in the .config file.

Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>
---
 Makefile |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/Makefile b/Makefile
index e40e7df..164788d 100644
--- a/Makefile
+++ b/Makefile
@@ -1154,6 +1154,10 @@ modules modules_install: FORCE
 	@echo
 	@exit 1
 
+PHONY += _modinst_
+_modinst_:
+	@echo "Modules disabled -- none to install"
+
 endif # CONFIG_MODULES
 
 ###
-- 
1.6.5.2

